lib/qrda-export/catIII-r2-1/qrda3_r21.rb

```diff
@@ -6,6 +6,8 @@ class Qrda3R21 < Mustache
 
   self.template_path = __dir__
 
+  PAYER_MAP = { '1' => 'A', '2' => 'B', '349' => 'D' }.freeze
```

Comment:
Aren't there hundreds of payer codes? How can we be sure that only these 3 will be used?